lib/system.nim

```diff
@@ -4210,6 +4210,23 @@ when hasAlloc or defined(nimscript):
       x[j+i] = item[j]
       inc(j)
 
+  func insert*(x: var string, c: char, i = 0.Natural) =
+    ## Inserts character `c` into `x` at position `i`
+    ##
+    ## .. code-block:: Nim
+    ##   var a = "hllo"
+    ##   a.insert('e', 1) # a <- "Hello"
+    var xl = x.len
```

Comment:
This `var` can be `let` I think. :thinking:

Reply:
True - I just copied the implementation from insert(string, string), so that is where this came from.